sql/system.go

```diff
@@ -103,6 +103,14 @@ var (
 		keys.ZonesTableID:      privilege.ReadWriteData,
 	}
 
+	systemTables = map[string]*TableDescriptor{
```

Comment:
Even if they are values, your brain would still have to realize that internal slices are shared with the `{Namespace,Descriptor,Lease,Users,Zones}Table` globals. Perhaps I'm not understanding the cognitive load the pointer is placing on you.